tests/e2e/test_create_view_edit_grant.py

```diff
@@ -8,20 +8,27 @@
 
 
 def test_create_view_edit_grant_success(
-    page: Page, domain: str, e2e_test_secrets: EndToEndTestSecrets, authenticated_browser: E2ETestUser
+    page: Page, domain: str, e2e_test_secrets: EndToEndTestSecrets, authenticated_browser_sso: E2ETestUser
 ):
     all_grants_page = AllGrantsPage(page, domain)
     all_grants_page.navigate()
 
     # Set up new grant
-    new_grant_page = all_grants_page.click_set_up_new_grant()
+    grant_intro_page = all_grants_page.click_add_a_grant()
+    grant_ggis_page = grant_intro_page.click_continue()
+    grant_ggis_page.select_no()
```

Comment:
🤏 nit:  We're about to make GGIS number compulsory, I think, so might be more interesting/valid to have this be yes and enter _some number_.